web/src/user_group_edit_members.js

```diff
@@ -36,9 +36,15 @@ function get_potential_members() {
     return people.filter_all_users(is_potential_member);
 }
 
-export function update_member_list_widget(group_id, member_ids) {
-    assert(group_id === current_group_id, "Unexpected group rerendering members list");
-    const users = people.get_users_from_ids(member_ids);
+function get_user_group_members(group) {
```

Comment:
This function should ideally be in `user_groups.ts`, but there were some lint issues due to two `User` types. I thought we can do this as a follow-up.

Reply:
OK